src/ontology/edits/oeo-physical.omn

```diff
@@ -5087,7 +5092,12 @@ Class: OEO_00030026
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "Primary energy production is a production that prepares raw material for its use as primary energy carrier."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000233> "add axioms:
+        <http://purl.obolibrary.org/obo/IAO_0000118> "primary production of energy",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "add eurostat synonym:
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000233> "add eurostat alternative term:
```
we agreed on `alternative term` instead of `synonym`